include/seqan/align/dp_formula_affine.h

```diff
@@ -59,30 +59,31 @@ namespace seqan {
 // Function _internalComputeScore      [RecursionDirectionDiagonal, AffineGaps]
 // ----------------------------------------------------------------------------
 
-template <typename TScoreValue, typename TAffineGaps, typename TTraceValueL, typename TTraceValueGap>
-inline SEQAN_FUNC_ENABLE_IF(Not<Is<SimdVectorConcept<TScoreValue> > >, typename TraceBitMap_<TScoreValue>::Type)
-_internalComputeScore(DPCell_<TScoreValue, TAffineGaps> & activeCell,
+template <typename TScoreValue, typename TTraceValueL, typename TTraceValueGap>
+inline SEQAN_FUNC_ENABLE_IF(Not<Is<SimdVectorConcept<TScoreValue> > >,
+                            typename TraceBitMap_<TScoreValue>::Type)
+_internalComputeScore(DPCell_<TScoreValue, AffineGaps> & activeCell,
                       TScoreValue const & rightCompare,
                       TTraceValueL,
                       TTraceValueGap,
```

Comment:
Probably the whole interface could be more readable. Sure. 
But I think this PR should not focus on renaming stuff. So I will reset all renamed variable names if there was no semantic reason for it.
We can adapt these interfaces when moving to SeqAn3.